source/hexo.md

```diff
@@ -35,8 +35,8 @@ title: 在Hexo 中使用
 ## hexo-theme-random
 > [Open](https://github.com/stiekel/hexo-theme-random/pull/64) `latest`
 
-## hexo-theme-material-x
-> [Supported](https://github.com/xaoxuu/hexo-theme-material-x/commit/05ba5ec30a95f4d05d69d76a99abf576304e4a6b) `latest`
+## hexo-theme-volantis
+> [Supported](https://github.com/xaoxuu/hexo-theme-volantis/) `latest`
```

Comment:
Please edit the [English document](https://github.com/xCss/Valine-docs/blob/master/source/en/hexo.md) at the same time, Thx!

Reply:
#36